_includes/payment-order-capture.md

```diff
@@ -8,12 +8,12 @@
 
 
 Capture can only be done on a payment with a successful authorized transaction,
-and if the authorization was not done on a one-phase payment instrument.
-Examples of one-phase payment instruments are [Swish][swish] and [Vipps][vipps],
- while payment instruments such as [Card Payments][card] are two-phase payments,
- requiring a `Capture` to be performed.
+and if the authorization was not done with a one-phase payment instrument.
+Examples of one-phase payment instruments are [Swish][swish] and
+[Trustly][trustly], while payment instruments such as [Card Payments][card] are
```

Comment:
Should we perhaps add `Vipps` as an example of two-phase payment instruments as well?

Reply:
Will do 👍